test/build.xml

```diff
@@ -0,0 +1,26 @@
+<project default="restart-iaf-test">
+	<target name="restart-iaf-test">
+		<exec executable="../../frank-runner/restart.bat" vmlauncher="false" failonerror="true">
+			<arg value="-Dproject.dir=iaf/test"/>
+			<!--
+			Optionally change application.server.type.custom as described in StageSpecifics_LOC.properties
+			Please note that ACTIVEMQ requires extra jar files that the Frank!Runner doesn't download yet, see:
+			https://github.com/ibissource/iaf/blob/master/TESTING_WITH_IAF-TEST.md#1-proprietary-modules-and-jar-dependencies
+			<arg value="-Dapplication.server.type.custom=ACTIVEMQ"/>
+			-->
+			<!-- Set .configurationFile and .basePath properties specified in DeploymentSpecifics.properties back to default values -->
+			<arg value="-Dconfigurations.MainConfig.configurationFile=Configuration.xml"/>
+			<arg value="-Dconfigurations.MainConfig.basePath=MainConfig"/>
+			<arg value="-Dconfigurations.HTTP.configurationFile=Configuration.xml"/>
+			<arg value="-Dconfigurations.HTTP.basePath=HTTP"/>
+			<arg value="-Dconfigurations.UTIL.configurationFile=Configuration.xml"/>
+			<arg value="-Dconfigurations.UTIL.basePath=UTIL"/>
+			<arg value="-Dconfigurations.TX.configurationFile=Configuration.xml"/>
+			<arg value="-Dconfigurations.TX.basePath=TX"/>
+			<arg value="-Dconfigurations.JDBC.configurationFile=Configuration.xml"/>
+			<arg value="-Dconfigurations.JDBC.basePath=JDBC"/>
+			<arg value="-Dconfigurations.JMS.configurationFile=Configuration.xml"/>
+			<arg value="-Dconfigurations.JMS.basePath=JMS"/>
```

Comment:
Ik vind het niet heel charmant dat je de configuratie van DeploymentSpecifics.properties hier nog eens over moet doen.
Kan dat niet anders?

Reply:
Als die regels uit de DeploymentSpecifics.properties worden verwijderd (zodat de framework defaults worden gebruikt) dan zijn deze regels ook niet in de build.xml nodig. Ik weet niet direct waarom er voor is gekozen het op deze manier in de DeploymentSpecifics.properties te zetten, wilde dat niet zomaar aanpassen zodat ook de bestaande werkwijze in Eclipse blijft werken. Maar nu ik er nog even naar kijk, misschien kunnen we het volgende in de pom.xml:

							<directory>src/main/configurations</directory>
							<targetPath>WEB-INF/classes/configurations</targetPath>

Aanpassen naar:

							<directory>src/main/configurations</directory>
							<targetPath>WEB-INF/classes</targetPath>

Dan zijn de regels in de DeploymentSpecifics.properties volgens mij niet meer nodig.

Het valt wel op dat niet voor alle configuraties hetzelfde is gedaan in de DeploymentSpecifics.properties. Sommige hebben een aangepast basePath en sommige niet. Is dat bewust om verschillende manieren te testen? En willen we dat blijven testen? Of is het gewoon niet zo consequent ingesteld en kunnen we het verwijderen (terug gaan naar defaults)?

Een andere optie zou zijn om de Frank!Runner de regels in de DeploymentSpecifics.properties te laten gebruiken maar dan worden de configuraties geladen uit het classpath (wat voor draaien op DEV wel handig is) maar dan moet je bij iedere config aanpassing een herstart doen. De Frank!Runner zorgt er zoals het nu is gedaan voor dat er directory classloaders worden gebruikt zodat je de config in je project aan kan passen en dan een reload in de console kan doen.